app/models/degree.rb

```diff
@@ -3,14 +3,22 @@
 class Degree < ApplicationRecord
   include Sluggable
 
+  INSTITUTIONS = Dttp::CodeSets::Institutions::MAPPING.keys
```

Comment:
Side note - would be nice to use this model defined constants in the `DegreesHelper` now.